public/app/features/alerting/unified/insights/SectionFooter.tsx

```diff
@@ -0,0 +1,17 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src/themes';
+import { useStyles2 } from '@grafana/ui';
+
+export function SectionFooter({ children }: React.PropsWithChildren<{}>) {
+  const styles = useStyles2(getStyles);
+
+  return <div className={styles.sectionFooter}>{children && <div>{children}</div>}</div>;
```

Comment:
I think this is unnecessary since `PropsWithChildren` has `children` defined as a `ReactNode` which can be either a JSX element or `null` and both can be rendered.